### PR TITLE
Ft user reset password 161779739

### PR DESCRIPTION
**Pull request by Oluwasegun-AA**
#### What does this PR do?
* Implement Password Reset Functionality
#### Description of Task to be completed?
* update `webpack.config.js` to allow the use of environment variables
* Add Password Reset View
* Add Appropriate Tests
* Add pull request template to aid raising PRs

#### How should this be manually tested?
- clone the repo using `git clone https://github.com/andela/helios-ah-frontend.git`
- `cd` into the `helios-ah-frontend` folder 
- checkout branch `ft-user-reset-password-161779739`
- add environment variables `PRODUCTION_URL='https://helios-ah-backend-staging.herokuapp.com/api/v1'` in the `.env` file
- run `npm install`
- run `npm build`
- run `npm start`
- visit `localhost:8000/reset-password` on your favourite web browser
#### Any background context you want to provide?
* N/A
#### What are the relevant pivotal tracker stories?
*  [#161779739]
#### Screenshots 
image

#### Questions:   
* N/A

### CODE REVIEW

File: jest.config.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "snapshotSerializers":[
+    "enzyme-to-json/serializer"
+  ]
+}
```
Comment (kingisaac95) [inline jest.config.json]: Newline here

File: src/styles/base/_base.scss [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
-  font-family: $reem-kufi;
+  font-family: $font-family;
   margin: 0;
   padding: 0;
 }
+
+#app, #app > div{
+  height:100%;
+  width: 100%;
+}
```
Comment (kingisaac95) [inline src/styles/base/_base.scss]: Newline here

Comment (Oluwasegun-AA) [reply src/styles/base/_base.scss]: Amended

File: jest.config.json (hunk 1 above)
Comment (Oluwasegun-AA) [reply jest.config.json]: resolved

**Review by kingisaac95 — APPROVED** [review]

**Pull request merged**